docs/developers-guide/partner-and-community-drivers.md

```diff
@@ -77,7 +77,7 @@ Anyone can build a community driver. These are the currently known third-party d
 | [Peaka](https://github.com/peakacom/metabase-driver)                                   | ![GitHub stars](https://img.shields.io/github/stars/peakacom/metabase-driver)                        | ![GitHub (Pre-)Release Date](https://img.shields.io/github/release-date-pre/peakacom/metabase-driver)                        |
 | [GreptimeDB](https://github.com/greptimeteam/greptimedb-metabase-driver)               | ![GitHub stars](https://img.shields.io/github/stars/greptimeteam/greptimedb-metabase-driver)         | ![GitHub (Pre-)Release Date](https://img.shields.io/github/release-date-pre/greptimeteam/greptimedb-metabase-driver)         |
 
-If you don't see a driver for your database, then try looking in the comments of the [issue related to the database](https://github.com/metabase/metabase/labels/Database%2F). You might also find more by [searching on GitHub](https://github.com/search?q=metabase+driver).
+If you don't see a driver for your database, then try looking in the comments of the [issue related to the database](https://github.com/metabase/metabase/labels/Database%2F). You might also find more by [searching on GitHub](https://github.com/search?q=metabase+driver&type=repositories).
```

Comment:
This link wasn't 404ing; we were getting a 429. So maybe we shouldn't link to it at all.

```suggestion
If you don't see a driver for your database, try looking in the comments of the [issue related to the database](https://github.com/metabase/metabase/labels/Database%2F). You might also find more drivers by searching on GitHub for "Metabase + driver".
```

Reply:
> This link wasn't 404ing; we were getting a 429. So maybe we shouldn't link to it at all.

True. This link is only accessible if you're logged in into GitHub.

Looks like it's a GitHub bug. Proof:
1. Log out of GitHub or use private browsing mode
2. Go to https://github.com/search
3. Type anything in the input and hit Enter

You're going to get Error 429.

-----

> So maybe we shouldn't link to it at all.

Sounds good :+1: 

> drivers by searching on GitHub for "Metabase + driver".

There will be no results when using `+` in the search query.
`+` appears in the URL because that's how GitHub encodes whitespace in the URL.

I'll just change it to:

```suggestion
If you don't see a driver for your database, try looking in the comments of the [issue related to the database](https://github.com/metabase/metabase/labels/Database%2F). You might also find more drivers by searching on GitHub for "Metabase driver".
```